operational/flexera/custom_branding/README.md

```diff
@@ -0,0 +1,75 @@
+# Custom Branding
+
+## What It Does
+
+This Policy allows a customer to brand the FlexeraOne platform quickly and consistently.
+
+## How It Works
+
+- Run the Custom Branding policy in the tenant ensuring you select the credentials required

```

Comment:
Please use correct grammar. For example, ending sentences in periods. Also, "policy template" instead of "policy" here

Reply:
Unresolved because issue was not addressed.